redis/src/commands/mod.rs

```diff
@@ -50,281 +63,7 @@ pub(crate) fn is_readonly_cmd(cmd: &[u8]) -> bool {
     )
 }
 
-macro_rules! implement_commands {
```

Comment:
I'd like this refactoring to be separated from the other changes in this PR, either as separate commits in this PR (with all the JSON changes squashed into a single PR) or as as a separate PR.